libnetwork/netavark/network.go

```diff
@@ -80,6 +83,9 @@ type InitConfig struct {
 	// DefaultsubnetPools contains the subnets which must be used to allocate a free subnet by network create
 	DefaultsubnetPools []config.SubnetPool
 
+	// DnsBindPort is set the the port to pass to netavark for aardvark
+	DnsBindPort int
```

Comment:
```suggestion
	DNSBindPort int
```

Reply:
Thanks I noticed these when looking at @Luap99 comments inline on github earlier (perhaps the linter warnings could also send a notification mail like normal reviews somehow?)

I think I fixed all of these now, will check when tests are done